README.md

```diff
@@ -88,6 +88,11 @@ Wrapped headers and replaced functions are:
     <td>Adds <code>clock_gettime</code> function (macOS10.11). Declares <code>asctime_r</code>, <code>ctime_r</code>, <code>gmtime_r</code>, and <code>localtime_r</code> functions that are otherwise hidden in the presence of <code>_ANSI_SOURCE</code>, <code>_POSIX_C_SOURCE</code>, or <code>_XOPEN_SOURCE</code> (OSX10.4)</td>
     <td>OSX10.4(11)</td>
   </tr>
+  <tr>
+    <td><code>time.h</code></td>
```

Comment:
This should be integrated into the prior entry (which is for `time.h`)

Reply:
@michaelld like this?